src/index.js

```diff
@@ -4,12 +4,23 @@ const electron = require("electron");
 const app = electron.app;
 const ipcMain = electron.ipcMain;
 const BrowserWindow = electron.BrowserWindow;
+const path = require('path');
+const fs = require("fs");
 let mainWindow;
 let subWindow;
 let initWindow;
 let isSubOpen = false;
 let isInitOpen = false;
 
+// loading window size and position
+const boundsFile = path.join(app.getPath('userData'), 'bounds.json');
```

Comment:
```suggestion
const boundsFilePath = path.join(app.getPath('userData'), 'bounds.json');
```

型が推測しづらい言語だからこそ、ファイル名なのかファイルバッファなのかわかるような命名をしたいなぁというお気持ちです。